client/cli/src/main.rs

```diff
@@ -43,6 +43,15 @@ use serde::{Deserialize, Serialize};
 use serde_json::{json, Value};
 use url::Url;
 
+/// List of modules supported by this cli
+fn get_module_gen_registry() -> ClientModuleGenRegistry {
+    ClientModuleGenRegistry::from(vec![
+        DynClientModuleGen::from(WalletClientGen),
+        DynClientModuleGen::from(MintClientGen),
+        DynClientModuleGen::from(LightningClientGen),
+    ])
+}
+
```

Comment:
After client NG lands we'll probably want to build `fedimint-cli` in a similar way to `fedimintd` with attaching modules?

Reply:
We were discussing with @justinmoon that we could have `fedimint-cli module <id/kind> <args>` and module itself would handle commands (`<args>`) to it.

Reply:
Yea. Ideally modules could define some CLI commands they would like to register to a `fedimint-cli` that includes as many modules as it likes ...

Reply:
Do we want to make `fedimint-cli` such a first-class citizen? Right now it's this hacky tool exposing whatever functionality CI needs.

Reply:
I think we want a "`fedimint-cli`" which is a first-class citizen, but might require something close to a rewrite once we have the new client?